### ISSUE TITLE
should the getting started application actually build without errors?

### ISSUE THREAD

**Issue by bvwelch** [IRNAS/irnas-east-software#75]
Greetings,

I am really interested in East -- I am hoping it will help with my zephyr development struggles.  But it seems like the zephyr example has moved on to use new features, and the getting started example no longer builds without errors.   And this sort of problem is one of my challenges -- I see interesting examples around on the internet, but it seems hard to match all of the pieces up such that I can actually build and test the example.   

Perhaps you could fork the zephyr example app, and use your fork to showcase EAST?
thank you.

**Comment by MarkoSagadin** [comment]
Hello @bvwelch, thank you for showing interest in the East :)

I have checked the example application repo. Current HEAD of the master is not compatible with any version of Zephyr. There were some changes in how the app versioning is done on their side and this does not yet seem to work, at least for me. 

I have updated the docs in the dev branch, so now they specify specific versions (see c34a29dad4477b for relevant changes). 

The example should build now, tell me if it does not.

**Comment by bvwelch** [comment]
sorry i did not intend to close this issue.

**Comment by MarkoSagadin** [comment]
Yep, the issue is that it does not find the `examplesensor0` node for the `nrf52840dk_nrf52840` board. I had solved this issue in the Release section, however the instructions were wrong.

I have updated the docs in the 1b726f78d3d122f18ba82ae6caceb29f8b320f8b
commit, try giving this a try.

If you get stuck I would recommend you to run `git clean -fd` from `example-application` directory and going through project setup steps again (only those that do some modification on the project, you do not need to run west init/update and such).
